azurelinuxagent/common/osutil/systemd.py

```diff
@@ -84,3 +84,39 @@ def get_unit_property(unit_name, property_name):
         raise ValueError("Can't find property {0} of {1}".format(property_name, unit_name))
     return match.group('value')
 
+
+def set_unit_property_run_time(unit_name, property_name, value):
+    """
+    Set a property of a unit at runtime
+
+    Note: --runtime settings only apply until the next reboot
+    """
+    try:
+        # Ex: systemctl set-property foobar.service CPUWeight=200 --runtime
+        shellutil.run_command(["systemctl", "set-property", unit_name, "{0}={1}".format(property_name, value), "--runtime"])
+    except shellutil.CommandError as e:
+        raise ValueError("Can't set property {0} of {1}: {2}".format(property_name, unit_name, e))
+
+
+def set_unit_properties_run_time(unit_name, properties):
+    """
+    Set multiple properties of a unit at runtime
+
+    Note: --runtime settings only apply until the next reboot
+    """
+    try:
+        # Ex: systemctl set-property foobar.service CPUWeight=200 MemoryMax=2G IPAccounting=yes --runtime
+        shellutil.run_command(["systemctl", "set-property", unit_name] + properties + ["--runtime"])
+    except shellutil.CommandError as e:
+        raise ValueError("Can't set properties {0} of {1}: {2}".format(properties, unit_name, e))
+
+
+def is_unit_loaded(unit_name):
+    """
+    Determine if a unit is loaded
```

Comment:
What does it mean for a unit to be loaded

Reply:
Unit being loaded means unit is parsed and available to the systemd process. We check this for extension services when we set quotas. If systemd is unaware of extension services and not loaded in the system yet, we get error while setting quotas. Hence, I added loaded check.

Reply:
`If systemd is unaware of extension services and not loaded in the system yet, we get error while setting quotas. Hence, I added loaded check.`

can you add this as a comment where you call this function?